packages/normalize-color/index.js.flow

```diff
@@ -7,4 +7,14 @@
  * @flow strict
  */
 
-declare module.exports: (color: ?(string | number)) => null | number;
+export type RgbaValue = {
+  +'display-p3'?: true,
+  srgb?: true,
```

Comment:
this is not good. These two should be mutually exclusive, the color space can be both srgb and display-p3.

Flow supports [enums](https://flow.org/en/docs/enums/using-enums/).
Can we create an enum `ColorSpace`?